app/views/hosts/_taxonomy_import_option.html.erb

```diff
@@ -1,2 +1,6 @@
+<label class="radio-inline">
   <%= f.radio_button :optimistic_import, "yes" %> <%= _("Fix %s on Mismatch") % taxonomy_type %>
```

Comment:
could you use https://github.com/theforeman/foreman/blob/develop/app/helpers/form_helper.rb#L99 instead? thanks!